users/views.py

```diff
@@ -44,6 +44,8 @@ def post(self, request):
         username = data.get('username')
         email = data.get('email')
         password = data.get('password')
+        allUsers = User.objects.values()  # retrieving all the values
```

Comment:
we don't want to fetch all the users on each register request since there could be a lot of users.  I think all you'd need to do is use `email = data.get('email').lower()` and that would prevent duplicate usernames from being created since we check it on line 56 already `if User.objects.filter(email=email).exists():`

Reply:
My bad. I will do it again. 
Should I amend the commit and make a force push or create a new pull request??

Reply:
Nice work!

Just fix it this bug and push it as a new commit. We don't need another PR.

Reply:
I have to do a force push. Hopefully, it's allright.

Reply:
Just make changes and then push! it won't need force push!